integration_tests/dbt_project.yml

```diff
@@ -0,0 +1,57 @@
+name: 'salesforce_integration_tests'
+version: '1.0.0'
```

Comment:
should this be 0.1.0?

Reply:
everything else looks goof to me!

Reply:
Just updated, thanks! Can you approve?

Reply:
(update to 0.3.0, which will be the next release of this package)